openpype/modules/default_modules/timers_manager/rest_api.py

```diff
@@ -48,3 +55,17 @@ async def start_timer(self, request):
     async def stop_timer(self, request):
         self.module.stop_timers()
         return Response(status=200)
+
+    async def get_task_time(self, request):
+        data = await request.json()
+        try:
+            project_name = data['project_name']
+            asset_name = data['asset_name']
+            task_name = data['task_name']
+        except KeyError:
+            log.error("Payload must contain fields 'project_name, " +
```

Comment:
It does not make much sense to log it only on server side. Rather send the message in response.
```suggestion
            message = "Payload must contain fields 'project_name, 'asset_name', 'task_name'"
            log.warning(message)
            return Response(text=message, status=404)
```